xrpl/wallet.go

```diff
@@ -26,8 +27,8 @@ type Wallet struct {
 
 // Creates a new random Wallet. In order to make this a valid account on ledger, you must
 // Send XRP to it.
-func NewWallet(alg addresscodec.CryptoImplementation) (Wallet, error) {
-	seed, err := keypairs.GenerateSeed("", alg)
+func NewWallet(alg addresscodec.CryptoImplementation, randomizer interfaces.Randomizer) (Wallet, error) {
```

Comment:
Why do we need to add the randomizer in the `NewWallet` function? 
I feel it would be simpler to place it inside the `NewWallet` function directly before line 31, don't you think?

As a developer I would just expect to write `xrpl.NewWallet(crypto.ED25519())`.
Or even `xrpl.NewWallet()` if we can make the default algorithm to ED25519.